src/core/Utils.js

```diff
@@ -401,6 +401,27 @@ function findDOMElement (element) {
   }
 }
 
+/**
+* Get metadata object from a form element
+*
+* @param {Node|string} element
+* @return {Object|null}
+*/
+
+function getMetaFromForm (element) {
+  if (!element || element.tagName !== 'FORM') {
+    console.error('Metadata can only be extracted from form elements')
+    return null
+  }
+
+  var formData = new FormData(element)
+  var result = {}
+  for (var entry of formData.entries()) {
```

Comment:
`entries()` is not supported in IE and Edge…but it looks like there isn't really another way to get data out of a FormData instance 😕 Maybe there's a library for that that doesn't use `FormData`?